mscore/plugin/api/qmlpluginapi.cpp

```diff
@@ -384,5 +384,10 @@ void PluginAPI::registerQmlTypes()
       qmlTypesRegistered = true;
       }
 
-}
-}
+void PluginAPI::sendScoreSaved(Ms::Score* s, bool successful, const QString& ext) // WARNING: Ms::Score is not the same as Ms::PluginAPI::Score!
+      {
+      emit scoreSaved(wrap<Score>(s), successful, ext);
```

Comment:
recursion prevention will be added here. Just trying to figure out how.
Tell me if you have any idea!